### PR TITLE
Adding orm

### PR DESCRIPTION
**Pull request by ManishaRana1195**
This helps in
1. being able to create and query tables with the help of ORM
2. beind able to manage migrations

### CODE REVIEW

File: app/src/main/java/com/geekskool/dhobi/VipasnaApplication.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+
+import android.app.Application;
+
+import com.geekskool.dhobi.Helpers.DbConstants;
+
+import io.realm.Realm;
+import io.realm.RealmConfiguration;
+
+public class VipasnaApplication extends Application {
```
Comment (deobald) [inline app/src/main/java/com/geekskool/dhobi/VipasnaApplication.java]: "vipassana"

File: app/src/main/java/com/geekskool/dhobi/VipasnaApplication.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+        Realm.init(this);
+        RealmConfiguration realmConfig = new RealmConfiguration.Builder()
+                .name(DbConstants.FILENAME)
+                .deleteRealmIfMigrationNeeded()
+                .schemaVersion(1)
```
Comment (deobald) [inline app/src/main/java/com/geekskool/dhobi/VipasnaApplication.java]: schema version = 0

**Review by deobald — CHANGES_REQUESTED** [review]
Make sure PR is atomic; bring fixes to ORM stuff in here.

**Comment by ManishaRana1195** [comment]
1. Did the requested changes.
2. Providing this [link](https://realm.io/docs/java/latest/#closing-realm-instances) and [this](https://realm.io/docs/java/latest/#controlling-the-lifecycle-of-realm-instances), implying that realm instances are not singleton.

**Comment by ManishaRana1195** [comment]
This pull request is associated with replacing the underlying SQlite Db with Realm.

**Pull request merged**